src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -152,23 +153,27 @@ def build_docs() {
 def upload_docs(name) {
     // upload the documentation
     // Takes the product name as an argument
+    doc_name = name.replace("_", "-")
```

Comment:
Please expand the docs to define a product name (should it have _ or - in the name? I am guessing "_" since you seem to use it to cd to directories at line 160).

Then perhaps note that this method replaces "_" with "-" for upload.

In the long run we may find packages for which that substitution is not enough, but at that point we can consider a fix, such as adding an optional doc_name argument to the main library entry point.